src/schema/v2/homeView/withHomeViewTimeout.ts

```diff
@@ -8,6 +8,14 @@ export const withHomeViewTimeout = (
   timeout: number = config.HOME_VIEW_RESOLVER_TIMEOUT_MS
 ): GraphQLFieldResolver<any, ResolverContext> => {
   return async (parent, args, context, info) => {
-    return await withTimeout(resolver(parent, args, context, info), timeout)
+    return await withTimeout(
+      resolver(parent, args, context, info),
+      timeout
+    ).catch((err) => {
+      const errorMessage = `[homeViewSection #${parent.id}]: ${err.message}`
```

Comment:
**suggestion (non-blocking):** I might tweak the format just a little…

Instead of…

```
[homeView section #home-view-section-new-works-for-you]: Timeout of 2000ms
```

I'd propose…

```
Timeout of 2000ms (home-view-section-new-works-for-you)
```

This way, we put the general info before the specific info, which I think is a more common pattern for error messages. And we can avoid some redundancy since the "home view section" info is already present in the section id, and still easily searchable (I assume) for DD purposes.

```suggestion
      const errorMessage = `${err.message} (${parent.id}): `
```